packages/website/src/content/01-03-touchpoint-ui-Core-Concepts.md

```diff
@@ -0,0 +1,169 @@
+- [Architecture](#architecture)
+  - [Core Components](#core-components)
+- [Key Concepts](#key-concepts)
+  - [Custom Components](#custom-components)
+  - [The ConversationHandler](#the-conversationhandler)
+- [Implementation Patterns](#implementation-patterns)
+  - [Pattern 1: Simple Chat Integration](#pattern-1-simple-chat-integration)
+  - [Pattern 2: Themed Chat with Custom Components](#pattern-2-themed-chat-with-custom-components)
+  - [Pattern 3: Programmatic Control](#pattern-3-programmatic-control)
+
+Touchpoint UI provides a powerful, flexible conversation interface that connects your application with your NLX conversational applications. Understanding the fundamental architecture and data flow will help you build more advanced implementations.
+
+## Architecture
+
+```mermaid
```

Comment:
@peterszerzo , can we add mermaid support _easily_? if so, that'd be great. Otherwise this is extra and can be rendered else where.

Reply:
@dtolb we'd have to look into it, probably!